server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -942,7 +942,7 @@ private CompletionStage<RestResponse> getCacheConfigMutableAttribute(RestRequest
    private CompletionStage<RestResponse> setCacheConfigMutableAttribute(RestRequest request) {
       NettyRestResponse.Builder responseBuilder = invocationHelper.newResponse(request);
       String attributeName = request.getParameter("attribute-name");
-      String attributeValue = request.getParameter("attribute-value");
+      String attributeValue = String.join(" ", request.parameters().get("attribute-value"));
```

Comment:
This seems like a pretty big change for other things that take parameters as well outside of this...

Reply:
`attribute.fromString()` already supports splitting space-separated multi-value attributes (this is how XML attributes work), so not really a big change.